main.go

```diff
@@ -0,0 +1,76 @@
+package main
+
+import (
+	partner "github.com/a-bigelow/cdk-eventbridge-partner-processors-go"
+	"github.com/aws/aws-cdk-go/awscdk/v2"
+	"github.com/aws/aws-cdk-go/awscdk/v2/awsevents"
+	"github.com/aws/constructs-go/constructs/v10"
+	"github.com/aws/jsii-runtime-go"
+)
+
+type ClusterStackProps struct {
+	awscdk.StackProps
+}
+
+func NewClusterStack(scope constructs.Construct, id string, props *ClusterStackProps) awscdk.Stack {
+	var sprops awscdk.StackProps
+	if props != nil {
+		sprops = props.StackProps
+	}
+	stack := awscdk.NewStack(scope, &id, &sprops)
+
+	// The code that defines your stack goes here
+	eventProps := awsevents.EventBusProps{EventBusName: jsii.String("name")}
+
+	eventBus := awsevents.NewEventBus(stack, jsii.String("id"), &eventProps)
+
+	secret := "VerySecretString"
```

Comment:
This should fulfill the `ISecret` protocol. `awscdk.Secret.fromSecretNameV2()` would fulfill this if you want to write up an example.

e.g. `awscdk.Secret.fromSecretNameV2(this, jsii.String("MyNuclearCodeSecret"), jsii.String("/home/recipes/icbm"))
`